test/test.js

```diff
@@ -272,6 +272,14 @@ describe('Page', function() {
     });
   });
 
+  describe('Browser.userAgent', function() {
+    it('should includes WebKit', async({browser}) => {
+      const userAgent = await browser.userAgent();
+      expect(userAgent.length).toBeGreaterThan(0);
+      expect(userAgent.includes('WebKit')).toBe(headless);
```

Comment:
`WebKit` is always a part of UA, so this should always be true

```js
expect(userAgent).toContain('WebKit');
```

Reply:
Done
https://github.com/GoogleChrome/puppeteer/pull/1716/commits/5565ed6c76b1926fedd031df06c9e1f824b3f5d7

Thank you for catching my mistake :)